### PR TITLE
Big-endian networking fixes for 1.5

### PR DESCRIPTION
**Pull request by glebm**

### CODE REVIEW

File: Source/dvlnet/frame_queue.cpp [hunk 1]
```diff
@@ -56,7 +57,7 @@ bool frame_queue::PacketReady()
 		if (Size() < sizeof(framesize_t))
 			return false;
 		auto szbuf = Read(sizeof(framesize_t));
-		std::memcpy(&nextsize, &szbuf[0], sizeof(framesize_t));
+		nextsize = LoadLE32(szbuf.data());
```
Comment (AJenbo) [inline Source/dvlnet/frame_queue.cpp:60]: Should we add an assert that checks that framesize_t is 32bit?`

Comment (glebm) [reply Source/dvlnet/frame_queue.cpp:60]: Done

**Comment by glebm** [comment]
Confirmed that it works!
Can join a game hosted by PPC and vice-versa! :tada:

**Comment by AJenbo** [comment]
Verified that it doesn't break anything :)

**Review by AJenbo — APPROVED** [review]

**Pull request merged**